apps/cowswap-frontend/src/modules/tokensList/pure/TokensVirtualList/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { useCallback, useMemo, useRef } from 'react'
+
+import { TokenWithLogo } from '@cowprotocol/common-const'
+
+import { useVirtualizer } from '@tanstack/react-virtual'
+import ms from 'ms.macro'
+
+import * as styledEl from './styled'
+
+import { SelectTokenContext } from '../../types'
+import { tokensListSorter } from '../../utils/tokensListSorter'
+import { CommonListContainer } from '../commonElements'
+import { TokenListItem } from '../TokenListItem'
+
+const estimateSize = () => 56
```

Comment:
Hum?
What about a const?

Reply:
It actually is :) The function is a const itself